compiler/damlc/tests/daml-test-files/List.daml

```diff
@@ -1,6 +1,15 @@
 -- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates.
 -- All rights reserved.
 
+-- @INFO Use null
+-- @INFO Evaluate
+-- @INFO Use sum
```

Comment:
You might need to add explicit location annotations to make this work. IIRC, something about them feels like being off by one.

Reply:
I avoided the off by one problem by not adding ranges. Doing the minimum possible to get the tests through.